recipes/doubtlab/meta.yaml

```diff
@@ -0,0 +1,114 @@
+{% set name = "doubtlab" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "e2065c916820ac6008883032594c14a0a660862cb012f3fac152a97f4c3c64b4" %}
+
+##----------------------------------------------------------------------##
+## TODO for Maintainers:
+##
+## >> After each PyPI release of the package (doublab):
+## 
+##    1. Get the corresponding "version" and "sha256" of the latest 
+##       release and update the jinja2 variables (line number: 2 and 3).
+##       - version
+##       - sha256
+##   
+##       Note: run the following commands from within the recipes/doubtlab. 
+##             $ . ./fetch_sha256.sh
+##             $ pypi_sha256 "doubtlab" "0.1.1" # this will give you the sha256  
+##
+##    2. Crosscheck and update the package dependency versions under:
+##       - requirements.host
+##       - requirements.run
+##
+## SEE THIS EXAMPLE: 
+## - how "pysteps" is maintaining "conda-forge/pysteps-feedstock":
+## https://pysteps.readthedocs.io/en/latest/developer_guide/update_conda_forge.html
+##
+##----------------------------------------------------------------------##
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/doubtlab-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - cleanlab >=1.0
+    - pandas >=1.3.3    
+    - scikit-learn >=1.0.0
+
+test:
+  imports:
+    - doubtlab
+  # commands:
+  #   - pip check
+  # requires:
+  #   - pip
```

Comment:
Is there a specific reason you're not using pip check? It's generally good practice to make sure that the reqs are up-to-date.

Reply:
Ok. I saw it was optional and so to keep things simple initially, I chose to comment out those lines. Will uncomment those lines and update the PR. Thank you.

Reply:
Yes, it's optional but it's just a nice extra "set of eyes". It doesn't take much CI time so it's fine to include it.